app/server/server.ts

```diff
@@ -44,7 +44,11 @@ const globals: Globals = {
 
 server.use(
   (req: express.Request, res: express.Response, next: express.NextFunction) => {
-    res.header("Cache-Control", "private");
+    // this header is VERY IMPORTANT and prevents caching (on both CDN and in browsers)
```

Comment:
I would address this in its own PR - it doesn't seem related to the Contributions tab to me?

Reply:
moved it to https://github.com/guardian/manage-frontend/pull/140 (which also nicely covers one the remediation Trello card)